### ISSUE TITLE
training about PIFuhd

### ISSUE THREAD

**Issue by lingtengqiu** [facebookresearch/pifuhd#112]
Thanks for your great contributions.

I am confused about training fine-pifu. I don't know whether I need to freeze the weights of coarse-pifu? If not, whether all loss need to add the loss of coarse-pifu? if yes, how to balance the weights between coarse loss and fine loss? 0.5 and 0.5?

**Comment by lingtengqiu** [comment]
through my experiments, I find freeze coarse-PIFu is better than training together.

**Comment by ljq2020** [comment]
Hi, Could you please share your training code? I met some problems when I train the pifuhd.

**Comment by hani1994a** [comment]
Hi, I have problem with training too.Could you please share your training code?
